kiva/compat.py

```diff
@@ -0,0 +1,23 @@
+"""
+Compatibility module to help support various versions of PIL and
+Pillow.
+
+"""
+import PIL
+from PIL import Image
+
+PILLOW_VERSION = getattr(PIL, 'PILLOW_VERSION', PIL.VERSION)
+
+
+def piltostring(image):
+    if PILLOW_VERSION.startswith('3'):
+        return image.tobytes()
```

Comment:
oops, that was wrong (sorry no coffee yet). So the issue is that the `tostring` is there but now raises
as a result we cannot just check for the attribute. Even better, the exception is `Exception` so a try except is going to be useless. So we have to check the Pillow version

*Note*: There is no enviroment variable